### PR TITLE
highlight reserved words

### PR DESCRIPTION
**Pull request by ayrat555**
fixes https://github.com/poanetwork/blockscout/issues/1820

### CODE REVIEW

**Comment by vbaranov** [comment]
@ayrat555 could you add a CHANGELOG entry?

**Comment by ayrat555** [comment]
@vbaranov  done

File: apps/block_scout_web/lib/block_scout_web/views/address_decompiled_contract_view.ex [hunk 1]
```diff
@@ -2,51 +2,231 @@ defmodule BlockScoutWeb.AddressDecompiledContractView do
...
+    "\e[32m" => "",
     # yellowgreen
-    "\e[93m" => "57, 115, 0",
+    "\e[93m" => "",
     # yellow
-    "\e[92m" => "119, 232, 81",
+    "\e[92m" => "",
     # red
-    "\e[94m" => "136, 0, 0"
+    "\e[94m" => ""
```
Comment (goodsoft) [inline apps/block_scout_web/lib/block_scout_web/views/address_decompiled_contract_view.ex]: Do I understand correctly that you almost completely disregard styles from decompiler now, except for gray color?

Comment (ayrat555) [reply apps/block_scout_web/lib/block_scout_web/views/address_decompiled_contract_view.ex]: yep

Comment (goodsoft) [reply apps/block_scout_web/lib/block_scout_web/views/address_decompiled_contract_view.ex]: What is gray anyway?

Comment (ayrat555) [reply apps/block_scout_web/lib/block_scout_web/views/address_decompiled_contract_view.ex]: comments

**Pull request merged**